crates/aranya-daemon/src/config.rs

```diff
@@ -29,11 +29,9 @@ pub struct Config {
     pub sync_addr: Addr,
 
     /// AFC configuration.
-    pub afc: AfcConfig,
+    pub afc: Option<AfcConfig>,
```

Comment:
```suggestion
    #[serde(default)]
    pub afc: Option<AfcConfig>,
```

And I guess remove it from our example configs?